pkg/operator/controller/garden/garden/components.go

```diff
@@ -186,6 +184,7 @@ func (r *Reconciler) instantiateComponents(
 	if err != nil {
 		return
 	}
+	// TODO(plkokanov): Remove this after gardener v1.109.0 has been released.
```

Comment:
Similar to https://github.com/gardener/gardener/pull/10796#discussion_r1838236028, we should be able to rely that HVPA CRD and CRs are already removed.